assets/templates/partials/calendar/filter/date.tmpl

```diff
@@ -1,4 +1,4 @@
-<form>
+<form  class="release-calender-released-date">
```

Comment:
One space between `form` and `class` please

Reply:
this is now done